composer.json

```diff
@@ -31,13 +31,16 @@
     "wpackagist-theme/twentytwenty": "^2.1"
   },
   "require-dev": {
+    "10up/debug-bar-elasticpress": "^2.1",
+    "10up/elasticpress": "^4.3",
```

Comment:
**suggestion:** I noticed that [VIP mu-plugins bundle](https://github.com/Automattic/vip-go-mu-plugins/blob/76e443bf76dab276346a1eacfb44ee5b281a3fb9/.gitmodules#L25-L27) their [own version](https://github.com/Automattic/ElasticPress) of ElasticPress -- should we rely on that instead?